doc/source/conf.py

```diff
@@ -183,6 +183,7 @@
     r"https://www.meetup.com/*",  # seems to be flaky
     r"https://www.pettingzoo.ml/*",  # seems to be flaky
     r"http://localhost[:/].*",  # Ignore localhost links
+    r"^http:/$",  # Ignore incomplete links
```

Comment:
why do we have incomplete links?

Reply:
this only happens in the docstring of `JobSubmissionClient` - linkcheck strips `"http://<head-node-ip>:8265"` to `http:/` and checks that (which obviously fails).